speaking-portal-project/src/animationFactory/ffmpeg.ts

```diff
@@ -0,0 +1,25 @@
+import { stdout, chdir, cwd } from 'node:process'
+import util from 'node:util'
+import { MouthCue, MouthCueArray } from '../types'
+
+const exec = util.promisify(require('node:child_process').exec)
+// TODO: Update parameters
+export async function ffmpegProcessor(audio_file_name: string, text_file_name: string) {
+  // Need child process verbose ? check out -> https://stackoverflow.com/questions/14332721/node-js-spawn-child-process-and-get-terminal-output-live
+  try {
+    chdir('../ffmpeg')
+  } catch (err) {
+    console.log(`Error message: ${err}`)
+  }
+  console.log(cwd())
+  // Run ffmpeg child process
+  // Warning: Command may not be correct for audio input
+  const { stderr } = await exec(`ffmpeg -f concat -i ${text_file_name} -i ${audio_file_name} -r 24 output.mp4`)
+
+  if (stderr) {
+    console.log(stderr)
+  }
+  //Todo: Return output
+
+  return
```

Comment:
Sweet! Not sure what the return would be...

Boolean that the video was created properly ?

Reply:
After downloading the latest version [2022-11-23 12:36](https://github.com/BtbN/FFmpeg-Builds/releases/tag/latest) for Windows, it looks like it's not an installation, just an .exe that will run similarly to Rhubarb, but I'll do a bit more testing first

Reply:
Once hooked into the API, I think the return could be the video itself potentially